src/native/libs/System.IO.Compression.Native/zlib_allocator_win.c

```diff
@@ -1,17 +1,16 @@
 // Licensed to the .NET Foundation under one or more agreements.
 // The .NET Foundation licenses this file to you under the MIT license.
 
+#define ZLIB_COMPAT 1
```

Comment:
This was a workaround I needed to add here because it didn't work when adding it to zlib-ng.cmake. What did I miss? It does not belong in here.